### PR TITLE
add push to push an item to stack

### PR DESCRIPTION
**Pull request by YuweAEC**
image

### CODE REVIEW

**Comment by gauravsinghaec** [comment]
Add 1 more option to list the stack items. Otherwise how will I know if your push is working or not?

File: daily-code/push-item.c [hunk 1]
```diff
@@ -0,0 +1,39 @@
+#include<stdio.h>
+#include<stdlib.h>
+#define ms 5
```
Comment (gauravsinghaec) [inline daily-code/push-item.c]: What is ms? give a meaningful name

File: daily-code/push-item.c [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+void push();
+int s[5], t=-1;
```
Comment (gauravsinghaec) [inline daily-code/push-item.c]: give a meaningful name e.g stack[5]

Comment (gauravsinghaec) [reply daily-code/push-item.c]: what is t? the name should be easy so that reader can understand the code without explanation

**Comment by gauravsinghaec** [comment]
I don't see any pop option in the code, how did u get the output?

File: daily-code/push-item.c (hunk 1 above)
Comment (YuweAEC) [reply daily-code/push-item.c]: okay changes made

File: daily-code/push-item.c (hunk 2 above)
Comment (YuweAEC) [reply daily-code/push-item.c]: given meaningful terms to ms and t

**Comment by YuweAEC** [comment]
> I don't see any pop option in the code, how did u get the output?

havn't  writtn pop function code

**Comment by YuweAEC** [comment]
> Add 1 more option to list the stack items. Otherwise how will I know if your push is working or not?
display function added in the code . for viewing the stack items

**Comment by YuweAEC** [comment]
> > I don't see any pop option in the code, how did u get the output?
> 
> haven't written pop function code

**Comment by gauravsinghaec** [comment]
Add an option to show the data. Currently you are printing after every push

**Review by gauravsinghaec — APPROVED** [review]

**Comment by YuweAEC** [comment]
> Add an option to show the data. Currently you are printing after every push

okay I will add that

**Pull request merged**